tests/cli/test_project.py

```diff
@@ -1,17 +1,37 @@
 from pathlib import Path
+import sys
 from tempfile import TemporaryDirectory
 from unittest.mock import MagicMock
 
 import pytest
 import readchar
+from typer import Exit
 import yaml
 from prefect.server import models
 from prefect.client import schemas
-from test_cloud import interactive_console  # noqa
 
 from prefect.testing.cli import invoke_and_assert
 
 
+@pytest.fixture
+def interactive_console(monkeypatch):
+    monkeypatch.setattr("prefect.cli.project.is_interactive", lambda: True)
+
+    # `readchar` does not like the fake stdin provided by typer isolation so we provide
+    # a version that does not require a fd to be attached
+    def readchar():
+        sys.stdin.flush()
+        position = sys.stdin.tell()
+        if not sys.stdin.read():
+            print("TEST ERROR: CLI is attempting to read input but stdin is empty.")
+            raise Exit(-2)
+        else:
+            sys.stdin.seek(position)
+        return sys.stdin.read(1)
+
+    monkeypatch.setattr("readchar._posix_read.readchar", readchar)
```

Comment:
@desertaxle I added this because the interactive tests pass locally even though they aren't running through the interactive steps (unless I use a cloud profile).

Thoughts on using this instead of the fixture from `test_cloud`?

Reply:
I think we should move this to a top-level fixture since we're using it in multiple test files, but this works for now and we can handle that in another PR.